torch/testing/_asserts.py

```diff
@@ -429,11 +383,18 @@ def append_difference(msg: str, *, type: str, difference: float, index: Tuple[in
 
     return msg.strip()
 
+def _get_XXX_dtype(dtype: torch.dtype) -> torch.dtype:
```

Comment:
_get_comparison_dtype?

Reply:
Add a comment for what this function does and how it's intended to be used

Reply:
Oops :innocent: